lib/phare/check/jshint.rb

```diff
@@ -25,7 +25,13 @@ def command
     protected
 
       def excluded_files
-        [] # TODO: Fetch the exclude list from .jshintignore
+        return [] unless configuration_file
+
+        configuration_file.split("\n").flat_map { |path| Dir.glob(path) }
```

Comment:
As @charlesdemers pointed out, event `jshint` is using a good old `split("\n")` ;)

https://github.com/jshint/jshint/blob/8e680e6accfac7c52933b4bc20a8b25c8382171a/src/cli.js#L200